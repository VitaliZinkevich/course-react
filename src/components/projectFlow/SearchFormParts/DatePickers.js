import React, { Component } from 'react';
import {Input} from 'react-materialize'


//event flow
import {mainFormFillEvents} from '../../../events/events'

class DatePickers extends  Component{
    
    handleChangeDates=(e, value)=>{
        mainFormFillEvents.emit('handleSearchForm', {name: e.target.name, value})
    }

    

render (){

    console.log('RENDER DATEPICKERS')
 

    // максимальная дата доступная в календаре
    let result = new Date()
    result.setDate(result.getDate()+183)
    
    let dateOptions = {
        min: new Date(),
        max: result,
        monthsFull: ['Январь', 'Февраль', 'Март', 'Апрель', 'Май', 'Июнь', 'Июль', 'Август', 'Сентябрь', 'Октябрь', 'Ноябрь', 'Декабрь'],
        weekdaysShort: ['Воскресенье', 'Понедельник', 'Вторник', 'Среда', 'Четверг', 'Пятница', 'Суббота'],
        monthsShort: ['Январь', 'Февраль', 'Март', 'Апрель', 'Май', 'Июнь', 'Июль', 'Август', 'Сентябрь', 'Октября', 'Ноября', 'Декабря'],
        weekdaysFull: ['Воск', 'Пон', 'Вт', 'Ср', 'Чтв', 'Птн', 'Суб'],
        firstDay: 1,
        showMonthsShort: true,
        showWeekdaysFull: true,
        format: 'dd.mm.yyyy',
        today: '',
        clear: 'Очистить',
        close: '',
        closeOnSelect: true,
    }

    
   return (
    <div>
        <Input 
        s={6} 
        value={this.props.valueFrom || ''}
        placeholder='Заселение с'
        name='dateFrom' 
        type='date' 
        onChange={(e, value)=>{this.handleChangeDates(e, value)}}
        options={dateOptions}
        
        />

        <Input
        s={6} 
        placeholder='Заселение по'
        name='dateTo' 
        type='date' 
        onChange={(e, value)=>{this.handleChangeDates(e, value)}} 
        options={dateOptions}
        value={this.props.valueTo || ''}
        />
    </div>
    )
    }
}

export default DatePickers
