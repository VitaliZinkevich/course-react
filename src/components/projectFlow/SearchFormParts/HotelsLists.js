import React, {PureComponent} from 'react'
import {CollapsibleItem, Collapsible,Row, Col} from 'react-materialize'

import HotelListItem from './HotelListItem'

import PropTypes from 'prop-types';
import ImmutablePropTypes from 'react-immutable-proptypes'

export class HotelsLists extends PureComponent {

    static propTypes={
        hotels: ImmutablePropTypes.listOf (ImmutablePropTypes.contains({
          _id: PropTypes.string.isRequired,
          name: PropTypes.string.isRequired,
          type: PropTypes.string.isRequired,
          stars: PropTypes.number.isRequired,
          rooms: ImmutablePropTypes.listOf(
              ImmutablePropTypes.contains ({
                  name:PropTypes.string.isRequired,
                  accomodation: ImmutablePropTypes.listOf(PropTypes.string).isRequired,
                  price: ImmutablePropTypes.map(
                      ImmutablePropTypes.contains({
                      adult:PropTypes.number,
                      children:PropTypes.number ,
                  }))
              })
    
          ),
      })),
      selectedHotels:ImmutablePropTypes.listOf (ImmutablePropTypes.contains({
        _id: PropTypes.string.isRequired,
        name: PropTypes.string.isRequired,
        type: PropTypes.string.isRequired,
        stars: PropTypes.number.isRequired,
        rooms: ImmutablePropTypes.listOf(
            ImmutablePropTypes.contains ({
                name:PropTypes.string.isRequired,
                accomodation: ImmutablePropTypes.listOf(PropTypes.string).isRequired,
                price: ImmutablePropTypes.map(
                    ImmutablePropTypes.contains({
                    adult:PropTypes.number,
                    children:PropTypes.number ,
                }))
            })
  
        ),
    }))
      }

 


  render() {
    //  console.log ('RENDER HOTEL LIST')
     //console.log (this.props)

     let collapsibleItemsMinsk = this.props.hotels.filter((el)=>{return(
        el.get ('region') ==='Минская область')}).map ((el)=>{
            return (
                
                <HotelListItem
                    key={el.get ('_id')}
                    hotel={el}
                    name='mainList'
                
                />
                 )
     } )

     let collapsibleItemsVitebsk = this.props.hotels.filter((el)=>{return(
        el.get ('region') ==='Витебская область')}).map ((el)=>{
            return (
               
                   <HotelListItem
                       key={el.get ('_id')}
                       hotel={el}
                       name='mainList'
                    />
           )
       
    } )
      

      let selectedHotels = this.props.selectedHotels.map ((element)=>{
          return (


            <HotelListItem
                key={element.get ('_id')}
                hotel={element}
                name='selectedList'
                
            />

        )

      })

    return (
      <div> 
            <Row>
                <Col s={8}>

                <h5>Список отелей</h5>
               
                    <Collapsible 
                    defaultActiveKey={0}
                    
                    >
                        <CollapsibleItem 
                        header='Минская область' 
                        icon='location_on'>
                            {collapsibleItemsMinsk}
                        </CollapsibleItem>
                    </Collapsible>

                    <Collapsible defaultActiveKey={0}
                    >
                        <CollapsibleItem  
                            header='Витебская область' 
                            icon='location_on'>
                            {collapsibleItemsVitebsk}
                        </CollapsibleItem>
                    </Collapsible>
                   
                </Col>

                <Col s={4}>
                        <div className='left'>
                        <h5>Выбранные отели</h5>
                        {selectedHotels}
                        </div>
                </Col>

            </Row>
      </div>
    )
  }
}

export default HotelsLists
