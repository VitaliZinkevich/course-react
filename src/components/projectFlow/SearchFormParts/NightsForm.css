 .nights {
    display: flex;
    flex-wrap: wrap;
    }
