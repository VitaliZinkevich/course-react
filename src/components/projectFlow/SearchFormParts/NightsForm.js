import React, { Component } from 'react';
import {Input, Navbar, NavItem, Button, Row} from 'react-materialize'


import {mainFormFillEvents} from '../../../events/events'

// + 1 ночь к расчетам


class NightsForm extends Component {
  
  handleNights=(e, ind)=>{
    mainFormFillEvents.emit ('handleSearchForm' , {name: e.target.name, value: ind+1})
  }

  render() {

    let startArray = []
    for (let i = 1; i!==22;  i++) {
      startArray.push(i)
    }

    let formView = startArray.map ((el, ind)=>{
      return (
        
        <div key={ind}>
        <Input
        checked={ind == 0} 
        name='nights' 
        type='checkbox' 
        labelClassName='black-text' 
        label={el} 
        onChange={(e)=>{this.handleNights(e,ind)}}/>
        </div>
        
      )
      

    })

      return (

      <div className='nights'>
        {formView}
      </div>
    );
  }
}

export default NightsForm;
