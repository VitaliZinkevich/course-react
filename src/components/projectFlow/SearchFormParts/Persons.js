import React from 'react'
import {Input, Navbar, NavItem, Button, Row, Icon} from 'react-materialize'

import {mainFormFillEvents} from '../../../events/events'

class Persons extends React.Component{

handlePersons=(e)=>{
    mainFormFillEvents.emit ('handleSearchForm' , {name: e.target.name,value:e.target.value})
}

render (){
    return (
        <div>
            
            <Input 
            icon='person'
            name ='adults'
            s={6} 
            type='select' 
            label="Adults" 
            labelClassName='black-text'
            onChange={this.handlePersons}>
                <option value='1'>1 Adult</option>
                <option value='2'>2 Adults</option>
                <option value='3'>3 Adults</option>
            </Input>

            <Input 
            name ='children'
            icon='child_care'
            s={6} 
            type='select' 
            label="Children" 
            labelClassName='black-text'
            onChange={this.handlePersons}>
                <option value='0'> 0 </option>
                <option value='1'> 1 Chield</option>
                <option value='2'> 2 Children</option>
                <option value='3'> 3 Children</option>
            </Input>
        </div>
            
        
    )
}

}

export default Persons