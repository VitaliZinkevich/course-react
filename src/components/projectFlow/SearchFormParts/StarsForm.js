import React, { PureComponent } from 'react'

import {Input} from 'react-materialize'
import {mainFormFillEvents} from '../../../events/events'

import PropTypes from 'prop-types';

export class StarsForm extends PureComponent {
    
    static propTypes={
        stars:PropTypes.string,
    }

  render() {
    let starsTypes = [ 'Любой','2','3','4','5']
    let startsOptions = starsTypes.map ((el,index)=>{
        return (
            <option key = {index} selected={starsTypes[index] === this.props.stars ? true: false} value={el}>{el}</option>
        )
    })

    return (
        <Input s={12} 
        onChange={(e)=>{
            mainFormFillEvents.emit ('handleSearchForm', {
                name: e.target.name,
                value:e.target.value})
        }}
        name='starsType'
        type='select' 
        label="Категория отеля"
        icon='star'
        >
        
            {startsOptions}
        </Input>
    )
  }
}

export default StarsForm
