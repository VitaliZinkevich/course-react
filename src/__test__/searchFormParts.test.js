import React from 'react';
import renderer from 'react-test-renderer';

// import parts

import NightsForm from '../components/projectFlow/SearchFormParts/NightsForm'
import Persons from '../components/projectFlow/SearchFormParts/Persons'
import DatePickers from '../components/projectFlow/SearchFormParts/DatePickers'
import HotelsLists from '../components/projectFlow/SearchFormParts/HotelsLists'
import StarsForm from '../components/projectFlow/SearchFormParts/StarsForm'
import FoodForm from '../components/projectFlow/SearchFormParts/FoodFrom'

import {fromJS} from 'immutable'

test('Snapshot test for search form parts', () => {

    const componentNightsForm = renderer.create(
      
      <NightsForm 
      valueNights={[1]}
      />
    );
  
    let componentNF=componentNightsForm.toJSON();
    expect(componentNF).toMatchSnapshot();
    
    const componentPersons = renderer.create(
        <Persons 
        childrenValue={0}
        adultValue={1}
        />
      );

    let componentP=componentPersons.toJSON();
    expect(componentP).toMatchSnapshot();

    const componentDatePickers = renderer.create(
      
        <DatePickers 
        valueFrom={null}
        valueTo={null}
        />
  
          
      );

    let componentDP=componentDatePickers.toJSON();
    expect(componentDP).toMatchSnapshot();
    
    let mockHotels = [{"_id" : "5b06859e5089ec123b9e668c",
                        "type" : "санаторий",
                        "name" : "Берестье",
                        "region" : "Минская область",
                        "stars" : 2,
                        "rooms": [{name:'Одноместный номер', accomodation:['1+0', '1+1'], price:{adult: 1000, children: 300} },
                                {name:'Двухместный номер', accomodation:['2+0', '2+1','2+2'], price:{adult: 1500, children: 400} },
                                {name:'Двухместный номер люкс', accomodation:['2+0', '2+1','2+2','3+0'], price:{adult: 1600, children: 500} }],
                        description: {
                            text:'санаторий Берестье (КУП «Брестагроздравница») расположен на территории курорта Белое озеро в Брестском районе. Санаторий находится в окружении массива сосново-лиственного леса уникального биосферного резервата и ландшафтного заказника «Прибужское Полесье». Территория санатория примыкает к озеру Рогознянское, которое соединяется каналом с озером Белое, Черное и далее Тайное. На территории санатория произрастают сосна, ель, береза, плодово-садовые деревья.',
                            fotos:['https://sanatorii.by/images/obj/89/c2598he5_433_true.jpg', 'http://belarus.svobodno.su/images/recovery/sanatoriy-bereste-0.jpg', 'https://i.ytimg.com/vi/h1TaZmN2waY/maxresdefault.jpg']
                        }

                        },
                        {"_id" : "5b06859e5089ec123b9e668f",
                        "type" : "санаторий",
                        "name" : "Белая вежа",
                        "region" : "Минская область",
                        "stars" : 3,
                        "rooms": [{name:'Одноместный номер с балконом', accomodation:['1+0', '1+1'], price:{adult: 1000, children: 300} },
                                    {name:'Двухместный двухкомнаный номер', accomodation:['2+0', '2+1','2+2'], price:{adult: 1500, children: 400} },
                                    {name:'Двухместный номер двухкомнаный номер люкс', accomodation:['2+0', '2+1','2+2','3+0'], price:{adult: 1600, children: 500} }]
                        }]

        mockHotels = fromJS(mockHotels)

    const componentHotelsLists = renderer.create(
      
        <HotelsLists 
        hotels={mockHotels}
        selectedHotels={[]}
        />
  
          
      );

    let componentHL=componentHotelsLists.toJSON();
    expect(componentHL).toMatchSnapshot();
   

    const componentStarsForm = renderer.create(
      
        <StarsForm 
         />
        );

    let componentSF=componentStarsForm.toJSON();
    expect(componentSF).toMatchSnapshot();


    const componentFoodForm = renderer.create(
      
        <FoodForm 
        foodValue={'Any'}
         />
        );

   
    let componentFF=componentFoodForm.toJSON();
    expect(componentFF).toMatchSnapshot();
    
    
  });

