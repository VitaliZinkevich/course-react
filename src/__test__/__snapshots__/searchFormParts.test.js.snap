// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for search form parts 1`] = `
<div
  className="nights"
>
  <div>
    <div
      className="col"
    >
      <input
        checked={true}
        className=""
        id="input_0"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_0"
      >
        1
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_1"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_1"
      >
        2
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_2"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_2"
      >
        3
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_3"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_3"
      >
        4
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_4"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_4"
      >
        5
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_5"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_5"
      >
        6
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_6"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_6"
      >
        7
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_7"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_7"
      >
        8
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_8"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_8"
      >
        9
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_9"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_9"
      >
        10
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_10"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_10"
      >
        11
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_11"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_11"
      >
        12
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_12"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_12"
      >
        13
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_13"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_13"
      >
        14
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_14"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_14"
      >
        15
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_15"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_15"
      >
        16
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_16"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_16"
      >
        17
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_17"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_17"
      >
        18
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_18"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_18"
      >
        19
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_19"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_19"
      >
        20
      </label>
    </div>
  </div>
  <div>
    <div
      className="col"
    >
      <input
        checked={false}
        className=""
        id="input_20"
        name="nights"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        className="black-text"
        htmlFor="input_20"
      >
        21
      </label>
    </div>
  </div>
</div>
`;

exports[`Snapshot test for search form parts 2`] = `
<div>
  <div
    className="col input-field s6"
  >
    <i
      className="material-icons prefix"
    >
      person
    </i>
    <label
      className="active black-text"
      htmlFor="input_21"
    >
      Взрослых
    </label>
    <select
      checked={false}
      className=""
      icon="person"
      id="input_21"
      name="adults"
      onChange={[Function]}
    >
      <option
        selected={true}
        value={1}
      >
        1
         взрослый
      </option>
      <option
        selected={false}
        value={2}
      >
        2
         взрослый
      </option>
      <option
        selected={false}
        value={3}
      >
        3
         взрослый
      </option>
    </select>
  </div>
  <div
    className="col input-field s6"
  >
    <i
      className="material-icons prefix"
    >
      child_care
    </i>
    <label
      className="active black-text"
      htmlFor="input_22"
    >
      Детей
    </label>
    <select
      checked={false}
      className=""
      icon="child_care"
      id="input_22"
      name="children"
      onChange={[Function]}
    >
      <option
        selected={true}
        value={0}
      >
        0
         ребенок
      </option>
      <option
        selected={false}
        value={1}
      >
        1
         ребенок
      </option>
      <option
        selected={false}
        value={2}
      >
        2
         ребенок
      </option>
      <option
        selected={false}
        value={3}
      >
        3
         ребенок
      </option>
    </select>
  </div>
</div>
`;

exports[`Snapshot test for search form parts 3`] = `
<div>
  <div
    className="col input-field s6"
  >
    <input
      checked={false}
      className=""
      id="input_23"
      name="dateFrom"
      onChange={[Function]}
      placeholder="Заселение с"
      type="date"
      value=""
    />
  </div>
  <div
    className="col input-field s6"
  >
    <input
      checked={false}
      className=""
      id="input_24"
      name="dateTo"
      onChange={[Function]}
      placeholder="Заселение по"
      type="date"
      value=""
    />
  </div>
</div>
`;

exports[`Snapshot test for search form parts 4`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col s8"
    >
      <h5>
        Список отелей
      </h5>
      <ul
        className="collapsible"
        data-collapsible="expandable"
      >
        <li
          className="active"
        >
          <div
            className="collapsible-header active"
            onClick={[Function]}
          >
            <i
              className="material-icons"
            >
              location_on
            </i>
            Минская область
          </div>
          <div
            className="collapsible-body"
          >
            <div
              className="hotelList"
            >
              <div
                className="col"
              >
                <input
                  checked={false}
                  className=""
                  id="input_25"
                  name="mainList"
                  onChange={[Function]}
                  type="checkbox"
                  value="5b06859e5089ec123b9e668c"
                />
                <label
                  className="active black-text"
                  htmlFor="input_25"
                >
                  Берестье
                </label>
              </div>
              <p>
                санаторий
              </p>
              <p>
                 
                <i
                  className="material-icons"
                >
                  star
                </i>
                 
                2
              </p>
            </div>
            <div
              className="hotelList"
            >
              <div
                className="col"
              >
                <input
                  checked={false}
                  className=""
                  id="input_26"
                  name="mainList"
                  onChange={[Function]}
                  type="checkbox"
                  value="5b06859e5089ec123b9e668f"
                />
                <label
                  className="active black-text"
                  htmlFor="input_26"
                >
                  Белая вежа
                </label>
              </div>
              <p>
                санаторий
              </p>
              <p>
                 
                <i
                  className="material-icons"
                >
                  star
                </i>
                 
                3
              </p>
            </div>
          </div>
        </li>
      </ul>
      <ul
        className="collapsible"
        data-collapsible="expandable"
      >
        <li
          className="active"
        >
          <div
            className="collapsible-header active"
            onClick={[Function]}
          >
            <i
              className="material-icons"
            >
              location_on
            </i>
            Витебская область
          </div>
          <div
            className="collapsible-body"
          />
        </li>
      </ul>
    </div>
    <div
      className="col s4"
    >
      <div
        className="left"
      >
        <h5>
          Выбранные отели
        </h5>
      </div>
    </div>
  </div>
</div>
`;

exports[`Snapshot test for search form parts 5`] = `
<div
  className="col input-field s12"
>
  <i
    className="material-icons prefix"
  >
    star
  </i>
  <label
    className="active"
    htmlFor="input_27"
  >
    Категория отеля
  </label>
  <select
    checked={false}
    className=""
    icon="star"
    id="input_27"
    name="starsType"
    onChange={[Function]}
  >
    <option
      value="Любой"
    >
      Любой
    </option>
    <option
      value="2"
    >
      2
    </option>
    <option
      value="3"
    >
      3
    </option>
    <option
      value="4"
    >
      4
    </option>
    <option
      value="5"
    >
      5
    </option>
  </select>
</div>
`;

exports[`Snapshot test for search form parts 6`] = `
<div>
   
  <div
    className="col input-field s12"
  >
    <i
      className="material-icons prefix"
    >
      free_breakfast
    </i>
    <label
      className="active"
      htmlFor="input_28"
    >
      Питание
    </label>
    <select
      checked={false}
      className=""
      icon="free_breakfast"
      id="input_28"
      name="foodType"
      onChange={[Function]}
    >
      <option
        selected={true}
        value="Any"
      >
        Любое
      </option>
      <option
        selected={false}
        value="AO"
      >
        Без питания
      </option>
      <option
        selected={false}
        value="BB"
      >
        Завтраки
      </option>
      <option
        selected={false}
        value="HB"
      >
        Завтрак и ужин
      </option>
      <option
        selected={false}
        value="FB"
      >
        Завтрак, обед и ужин
      </option>
      <option
        selected={false}
        value="ALL"
      >
        Все включено
      </option>
    </select>
  </div>
</div>
`;
